kf_model_fhir/ingest_plugin/target_api_builders/kfdrc_condition.py

```diff
@@ -0,0 +1,99 @@
+        assert None is not record[CONCEPT.PARTICIPANT.ID]
+        assert None is not record[CONCEPT.DIAGNOSIS.NAME]
+        return record.get(CONCEPT.DIAGNOSIS.UNIQUE_KEY) or join(
+            record[CONCEPT.PARTICIPANT.ID],
+            record[CONCEPT.DIAGNOSIS.NAME],
+            record.get(CONCEPT.DIAGNOSIS.EVENT_AGE_DAYS),
+        )
+
+    @staticmethod
+    def build_entity(record, key, get_target_id_from_record):
+        study_id = record[CONCEPT.STUDY.ID]
+        diagnosis_name = record.get(CONCEPT.DIAGNOSIS.NAME)
+        event_age_days = record.get(CONCEPT.DIAGNOSIS.EVENT_AGE_DAYS)
+        icd_id = record.get(CONCEPT.DIAGNOSIS.ICD_ID)
+        mondo_id = record.get(CONCEPT.DIAGNOSIS.MONDO_ID)
+        ncit_id = record.get(CONCEPT.DIAGNOSIS.NCIT_ID)
+
+        entity = {
+            "resourceType": Condition.resource_type,
+            "id": get_target_id_from_record(Condition, record),
+            "meta": {
+                "profile": [
+                    "http://fhir.kids-first.io/StructureDefinition/kfdrc-condition"
+                ]
+            },
+            "identifier": [
+                {
+                    "system": "urn:kids-first:unique-string",
+                    "value": join(Condition.resource_type, study_id, key),
+                }
+            ],
+            "category": [
+                {
+                    "coding": [
+                        {
+                            "system": "http://terminology.hl7.org/CodeSystem/condition-category",
+                            "code": "encounter-diagnosis",
+                            "display": "Encounter Diagnosis",
+                        }
+                    ]
+                }
+            ],
+            "code": {"text": diagnosis_name},
+            "subject": {
+                "reference": f"Patient/{get_target_id_from_record(Patient, record)}"
+            },
+        }
+
+        if event_age_days:
+            entity.setdefault("extension", []).append(
+                {
+                    "url": "http://fhir.kids-first.io/StructureDefinition/age-at-event",
+                    "valueAge": {
+                        "value": int(event_age_days),
+                        "unit": "d",
+                        "system": "http://unitsofmeasure.org",
+                        "code": "days",
+                    },
+                }
+            )
+
+        if icd_id:
+            entity["code"].setdefault("coding", []).append(
+                {"system": "http://hl7.org/fhir/sid/icd-10", "code": icd_id}
+            )
+
+        if mondo_id:
+            entity["code"].setdefault("coding", []).append(
+                {
+                    "system": "http://purl.obolibrary.org/obo/mondo.owl",
+                    "code": mondo_id,
+                }
+            )
+
+        if ncit_id:
+            entity["code"].setdefault("coding", []).append(
+                {"system": "http://ncit.nci.nih.gov", "code": ncit_id}
```

Comment:
Can you replace the system URL with `"http://purl.obolibrary.org/obo/ncit.owl"`?

Reply:
Why though? That's different than the URI format for ncimeta @ https://www.hl7.org/fhir/terminologies-systems.html and also the OID registry per https://wiki.nci.nih.gov/pages/viewpage.action?pageId=7472532

Reply:
I think at best it would be either as is (per "Source Home Page: https://ncit.nci.nih.gov" on https://ncit.nci.nih.gov/ncitbrowser/) or "urn:oid:2.16.840.1.113883.3.26.1.1".

Reply:
We talked about it during the standup and I repeat my rationale here: in terms of the official website URL, the existing one in the plugin as well as the above two @fiendish refers to are legitimate. As a resource, I figured the owl one is official by obo (For example, Javacript libraries may have their own official websites, and are also deposited to CDN as permanent resources). So, as Avi mentioned, what we should for a system URL seems not very well defined (even by FHIR). I am very open to further discussion.

Reply:
Also, the canonical OWL file URL is http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl per https://confluence.hl7.org/display/TA/National+Cancer+Institute+-+Thesaurus

Reply:
Let's keep it as it is for now.

Reply:
So where does that leave us?